### DISCUSSION TITLE
Dex image is showing as Vulnerability in S360

### DISCUSSION THREAD

**Discussion by chaitutheprince** [dexidp/dex#2654] [Q&A]
Hi All

We have pulled Laster version of Kots 1.83. As per discussion with one of the engineer in kots,  busybox CVE was fixed in v2.33.0 and this includes in latest version of kots. Post pushing that Image to our container, it shows Sha value as d358430c1fd0b755632058eca89b5093550ca979ce3b01b5a79f63ed89fbd500. In our S360 vulnerability, we see still the image is vulnerable.

Few minutes before I tried to pull the image from ghcr.io/dexidp/dex:v2.33.0 to our container. now it shows Sha value as 62902bd3a7ce4a73ed44e28884bffec04bcaa4a07e31e043173bdce289717e80.

Can you confirm us which image clears the vulnerability 

vulnerability ID = 502444
Vulnerability name =-Alpine Linux Security Update for busybox

![image](https://user-images.githubusercontent.com/111914825/188937376-a77e75f8-6662-44ce-b7e7-6d9790158447.png)

**Comment by sagikazarmark** [comment]
Can you please check if upgrading alpine helped?

https://github.com/dexidp/dex/releases/tag/v2.33.1

**Reply by chaitutheprince** [reply]
It's cleared now. Thanks @sagikazarmark.
